edk2toollib/gitignore_parser.py

```diff
@@ -179,7 +179,13 @@ def match(self, abs_path: str) -> bool:
         """Returns True or False if the path matches the rule."""
         matched = False
         if self.base_path:
-            rel_path = _normalize_path(abs_path).relative_to(self.base_path).as_posix()
+            try:
+                rel_path = _normalize_path(abs_path).relative_to(self.base_path).as_posix()
+            except ValueError as e:
```

Comment:
Please write a test for this, to ensure the functionality is not accidently lost :)

Reply:
Added